compiler/lquantizer/include/QuantizationObservers.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LQUANTIZER_QUANTIZATION_OBSERVERS_H__
+#define __LQUANTIZER_QUANTIZATION_OBSERVERS_H__
+
+#include <luci_interpreter/Interpreter.h>
+#include <luci_interpreter/core/Tensor.h>
+
+#include <vector>
+#include <map>
+
+namespace lquantizer
+{
+
+using Node2NodeMatcher = std::map<const luci::CircleNode *, const luci::CircleNode *>;
+using Node2InputDataMatcher = std::map<const luci::CircleNode *, std::vector<float>>;
```

Comment:
Let's use `unordered_map` latter.

Reply:
We use vector as data storage because position of value in input tensor no matter for as, we need to minimize overall error!